### ISSUE TITLE
Sorry to open another issue ticket but:

### ISSUE THREAD

**Issue by capjay45** [RestoreMonarchyPlugins/Pickpocket#2]
I gave the mafia permissions bypass.pickpocket and joined mafia job, but I can't pickpocket people. Have I done something wrong?

**Comment by RestoreMonarchy** [comment]
No it should work unless they are mafia too, are you sure you're pointing at them close enough?

**Comment by capjay45** [comment]
> No it should work unless they are mafia too, are you sure you're pointing at them close enough?

yes but it wont even say anything in chat.
edit: I am right behind them and they are completely still. it doesnt say that i dont have permissions or anything. it says nothing

**Comment by RestoreMonarchy** [comment]
and you're sure it was working when you didn't use the bypass permission?

**Comment by capjay45** [comment]
> and you're sure it was working when you didn't use the bypass permission?

it wasn't working before I gave them that permission either. Did I put in the wrong command? I just read that bypass.pickpocket makes you invincible to being pickpocketed. so what is that permission to make people able to pickpocket?

**Comment by RestoreMonarchy** [comment]
Oh they need `pickpocket` permission if you enabled the UsePermissions ;P Sorry I missed that

**Comment by capjay45** [comment]
> Oh they need `pickpocket` permission if you enabled the UsePermissions ;P Sorry I missed that

Oh ok thanks. So I just have to do "<Permission Cooldown="0">pickpocket</Permission>" yes?

**Comment by RestoreMonarchy** [comment]
yeah give `pickpocket` permission to the group you want to be able to pickpocket

**Comment by capjay45** [comment]
> yeah give `pickpocket` permission to the group you want to be able to pickpocket

ok Ill try that

**Comment by capjay45** [comment]
> yeah give `pickpocket` permission to the group you want to be able to pickpocket

It works. Thanks you legend.
